browser/test/integration/uiSlotValues.js

```diff
@@ -0,0 +1,68 @@
+var assert = require('assert');
```

Comment:
would be good to add tests for:
- update_input() gets called with default values before first update_state()